slack-first/base/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: slack-first
+  labels:
+    app: slack-first
+spec:
+  selector:
+    matchLabels:
+      app: slack-first
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: slack-first
+    spec:
+      containers:
+        - name: slack-first
+          image: quay.io/tcoufal/slack-first:latest
```

Comment:
Is it too early to move the image over to the operate-first umbrella?

Reply:
I think the devel on it is not happening in `operate-first` umbrella - I have my own CI set up on this and I'm releasing to quay from GitHub Actions during A GH release process (see here for example: https://github.com/tumido/slack-first/issues/29). I probably don't want to be releasing to operate-first from a my private (yet public) repo.